TheMovieDb/Common/LoadingViewController.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  LoadingViewController.swift
+//  TheMovieDb
+//
+//  Created by Ricardo Ramirez on 02/11/21.
+//
+
+import UIKit
+
+class LoadingViewController: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let spinner = UIActivityIndicatorView()
+        spinner.translatesAutoresizingMaskIntoConstraints = false
+        spinner.startAnimating()
+        view.addSubview(spinner)
+        
+        NSLayoutConstraint.activate([
+            spinner.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            spinner.centerYAnchor.constraint(equalTo: view.centerYAnchor)
+        ])
```

Comment:
Let's keep clean the viewDidLoad. You can make functions to this responsibilities.

Reply:
Solved